### PR TITLE
:rocket: Big-chunck-codes of `Todo-list` project.

### PR DESCRIPTION
**Pull request by mahabubx7**
### :rocket: Big-chunck-codes of `Todo-list` project.
:heavy_check_mark: No linters errors.
:heavy_check_mark: Good GitHub workflows.
:heavy_check_mark: Complete project.

### CODE REVIEW

File: src/libs/Renderer.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  // eslint-disable-next-line class-methods-use-this
+  serialUpdates() {
+    const updated = [];
+    const currentList = document.querySelectorAll('#list li');
+    // eslint-disable-next-line no-restricted-syntax, guard-for-in
+    for (const it in currentList) {
+      updated.push(this.list[Number(currentList[it].dataset?.index)]);
+    }
+    // updated = updated.slice(0, currentList.length);
```
Comment (Jeddaa) [inline src/libs/Renderer.js]: leaving old codes as inline comments are not advisable so as not to make your code look messy.

File: server.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const express = require('express');
```
Comment (Jeddaa) [inline server.js]: I don't think this is required for github pages.

**Review by Jeddaa — COMMENTED** [review]
giphy

ALmost there!
This is a great work and just requires very little adjustment to follow the javascript best practices.
Welldone!

**Comment by mahabubx7** [comment]
@Jeddaa Thanks for the review! :+1: 
Now, you can re-review it since I have made your required changes.
:man_technologist:

**Review by Jeddaa — APPROVED** [review]
clapping-leonardo-dicaprio
Welldone! There is nothing left to do. You can now merge.

**Pull request merged**